prime-router/metadata/schemas/covid-19.schema

```diff
@@ -890,6 +890,13 @@ elements:
     documentation: |
       The name and OID for the application sending information to the receivers
 
+  - name: site_of_care
+    type: TEXT
+    natFlatFileField: Site_of_care
+    hhsGuidanceField:
+    hl7Field:
```

Comment:
remove hl7Field since you are not specifying it.  I would remove any other fields that do not have values.